api/project.py

```diff
@@ -41,3 +64,62 @@ def get(self, request) -> JSONResponse:
             "knowledge_base_ids": [str(list.id) for list in project.knowledge_bases],
         }
         return JSONResponse(result)
+
+
+class ProjectCreationFromWorkflow(HTTPEndpoint):
```

Comment:
much of what happens here would be outsourced to another layer in our other services. Request managment (reading data out of request and auth of user) can of course be done in an api, but application logic like checking and creating a bucket or session management of a database could be put into a controller file, at least thats what i would have expected

Reply:
where applicable, i transferred it into the controller logic of project (`add_workflow_store_data_to_project`)